tests/test_expectations.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import pytest
+
+from sgs.expectations import expectations
+
+
+@pytest.mark.expectations
+def test_expectations_monthly():
+    actual = expectations("monthly", "01/01/2020", "01/02/2020").shape
+    expected = (4668, 11)
+
+    assert actual == expected
+
+
+@pytest.mark.expectations
+def test_expectations_invalid_resource():
+    actual = expectations("foobar", "01/01/2020", "01/02/2020")
+    expected = None
+
+    assert actual == expected

```

Comment:
Alterar o teste test_expectations_invalid_resource. A função, quando alimentada com um resource inválido, retorna um ValueError

```suggestion
def test_expectations_invalid_resource():
    with pytest.raises(ValueError):
        expectations("foobar", "01/01/2020", "01/02/2020")
```